docs/product_release.md

```diff
@@ -0,0 +1,72 @@
+Product releases are done by creating a GitHub release. Every release is following the [Semantic Versioning](https://semver.org/) pattern and must be marked in the git history via tag.
```

Comment:
I'd suggest to add a title (top level heading) to align to other docs
```
# Product Release
## Intro
```

Reply:
done